specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +860,98 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDnsZoneGroup": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "type": {
```

Comment:
is type section needed at all?

Reply:
See VirtualNetworkPeering from here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetwork.json

Reply:
Also do you need etag?

Reply:
"type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups was what I saw in the example JSON file in the groups object, 

(Full object; {
    "name": "dnsgroup1",
    "id": "/subscriptions/a9c98ed2-1a82-446b-9ef6-dbb935df6614/resourceGroups/group 1/providers/Microsoft.Network/privateEndpoints/pe_zone/privateDnsZoneGroups/dnsgroup1",
    "type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups",
    "properties": {
)

I could change this if need be.

Reply:
done pushing when im done resolving other comments